### DISCUSSION TITLE
Migrating from Jenkins to GitHub Actions

### DISCUSSION THREAD

**Discussion by sharkudev** [community/community#67063] [Actions]
### Select Topic Area

Question

### Body

Jenkins has some advantages over GitHub Actions, such as a larger community and support for a wider range of plugins.
    GitHub actions have some limitations in the features they are supporting for example, [the inputs view](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch) is not customizable, and the fields functionality is not dynamic.
    Dependency on GitHub: Using GitHub Actions requires that your code and repositories be hosted on GitHub. If you use other Git hosting services or have private repositories that you don’t want to host on GitHub, then GitHub Actions may not be the best choice for your organization.
    When using GitHub actions you will be more exposed to instability in terms of infrastructure (especially if you will use Github actions runners) and in general the Actions [services](https://www.githubstatus.com/).
    Security: While GitHub Actions has built-in security features, there have been concerns raised about the security of workflows that execute code from external sources, you’ll need to dig into Github Action [best practices](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions). Jenkins, on the other hand, has been around for a long time and has a proven track record when it comes to security.

**Comment by fardinahosancse** [comment]
Read this : https://medium.com/@yarindeoh/a-journey-for-migrating-jenkins-to-github-actions-4fc635f541d4

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
